core/imodel-bridge/src/test/standalone/iModelBridgeFwkStandAlone.test.ts

```diff
@@ -46,6 +47,9 @@ describe("IModelBridgeFwkStandAlone", () => {
     const runner = new BridgeRunner(bridgeJobDef);
     const fileName = `${path.basename(assetFile, path.extname(assetFile))}.bim`;
     const filePath = path.join(KnownTestLocations.outputDir, fileName);
+    const issueReporter = new BADGERSIssueReporter("", "", "", "TestBridge.json", undefined, assetFile, KnownTestLocations.outputDir);

```

Comment:
Need to update this with relevant context, activityId and iModelId (may not be present in standalone)